repos/spack_repo/builtin/packages/scorep/package.py

```diff
@@ -107,21 +108,26 @@ def url_for_version(self, version):
         description="Enable debug info lookup via binutils",
         when="^binutils",
     )
+    variant("fortran", default=True, description="Enable fortran support")
     # Putting this in as preparation. F08 support exists in 9.0 but configure does not respect
     # --enable-mpi-f08 and will not until 9.1.
-    variant("mpi_f08", default=True, description="Enable MPI F08 support", when="@9.1: +mpi")
+    variant(
+        "mpi_f08", default=True, description="Enable MPI F08 support", when="@9.1: +mpi +fortran"
+    )
+    variant(
+        "gotcha", default=True, description="Enable library wrapping with libgotcha", when="@9.0:"
+    )
     # Dependencies for SCORE-P are quite tight. See the homepage for more
     # information. Starting with scorep 4.0 / cube 4.4, Score-P only depends on
     # two components of cube -- cubew and cubelib.
 
     # Language dependencies
-    # TODO: we could allow a +fortran variant here.
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("fortran", type="build", when="+fortran")  # generated
```

Comment:
Please remove the comment (`# generated`) if the language dependencies are confirmed.

Reply:
Good question. That is generally the case with dependencies but isn't the instrumentation added during the build?